source/tests/canpie-fd/test_cp_msg_ccf.c

```diff
@@ -28,29 +28,52 @@
 
 
 /*----------------------------------------------------------------------------*\
-** Pre-condition                                                              **
+** Pre Condition files                                                        **
 **                                                                            **
-\*----------------------------------------------------------------------------*/
+\*---------------------------------------------------------------------------*/
 
+/*!
+** \def  CP_CAN_FD
+** ** This symbol enables or disables the CAN FD extension
+** \def CP_CAN_MSG_MACRO
+** This symbol enables or disables using MSG_MACROs or MSG_FUNCTIONs.
+*/
 #define  CP_CAN_FD               0
 #define  CP_CAN_MSG_MACRO        0
 
 /*----------------------------------------------------------------------------*\
 ** Include files                                                              **
 **                                                                            **
 \*----------------------------------------------------------------------------*/
-
 #include "cp_msg.h"
 #include "unity_fixture.h"
 
+#include <stdio.h>
+#include <string.h>
+
+//----------------------------------------------------------------------------//
+/*!
+** \file    test_cp_msg_ccf.c
```

Comment:
Code style has to be observed:
** \file    test_cp_msg_ccf.c
** \brief   Test cases for message access functions
**
** The following test cases cover the CANpie FD message API. They are
** described inside the CANpie FD test specification and marked as
** CP_MSG_CCF_<num>.
** <p>